src/current.ts

```diff
@@ -178,7 +182,8 @@ export async function readCurrentMigration(
     if (headerLines.length) {
       wholeBody = headerLines.join("\n") + "\n\n" + wholeBody;
     }
-    return wholeBody;
+
+    return await compileIncludes(_parsedSettings, wholeBody);
```

Comment:
`return await` is unnecessary (and consumes an extra tick) unless you're inside a `try{}` block.

```suggestion
    return compileIncludes(_parsedSettings, wholeBody);
```